onebusaway-gtfs-transformer/src/main/java/org/onebusaway/gtfs_transformer/impl/CheckForPlausibleStopTimes.java

```diff
@@ -49,15 +51,39 @@ public void run(TransformContext context, GtfsMutableRelationalDao dao) {
                     oldTime = newTime;
                 }
                 //check if the bus takes more than five hours between stops
-                if(newTime.getArrivalTime() - oldTime.getDepartureTime() > 60*60*5){
-                    _log.error("Trip {} on Route {} is scheduled for unrealistic transit time between {} at {}, and {} at {}", trip.getId(), trip.getRoute(), oldTime.getId(), oldTime.getDepartureTime(), newTime.getId(), newTime.getDepartureTime());
-                    //es.publishMessage(getTopic(), "Trip " + trip.getId() + " on Route "+ trip.getRoute() +" is scheduled for unrealistic transit time between " + oldTime.getId()+ " at " + oldTime.getDepartureTime() + ", and " +  newTime.getId() + " at " + newTime.getDepartureTime()););
+                if(newTime.getArrivalTime() - oldTime.getDepartureTime() > 5 * MINUTES_PER_HOUR * SECONDS_PER_MINUTE){
+                    _log.error("Trip {} on Route {} is scheduled for unrealistic transit time between trip {} at {}, and trip {} at {}", trip.getId(), trip.getRoute(), oldTime.getId(), humanReadableTime(oldTime.getDepartureTime()), newTime.getId(), humanReadableTime(newTime.getArrivalTime()));
+                    es.publishMessage(getTopic(), "Trip " + trip.getId() + " on Route "+ trip.getRoute() +" is scheduled for unrealistic transit time when traveling between stoptime" + oldTime.getId()+ " at " + oldTime.getDepartureTime() + ", and stoptime" +  newTime.getId() + " at " + newTime.getDepartureTime());
```

Comment:
When you find yourself concatenating many sequences of strings either consider a string buffer or string formatting via message building: https://docs.oracle.com/javase/7/docs/api/java/text/MessageFormat.html

Reply:
My understanding is that for performance going forward (Java 9 and higher) it's actually preferred to use the `+` operator:
https://dzone.com/articles/jdk-9jep-280-string-concatenations-will-never-be-t

Although let me know if I'm wrong :).

Reply:
Either way I might as well not be writing the same message twice..